README.md

```diff
@@ -35,7 +35,13 @@ Installation is straightforward.  Pwndbg is best supported on Ubuntu 18.04 with
 ```shell
 git clone https://github.com/pwndbg/pwndbg
 cd pwndbg
-./setup.sh
+./setup.sh virtualenv
+```
+
+Then to use pwndbg do:
+
+```
+source pwndbg/fake_env/bin/activate
```

Comment:
Btw wouldn't it be better to call it `venv` instead of `fake_env`?

Also, I am not sure if its ok that we depend on `virtualenv` being already installed; we should probably say it

Reply:
done